### ISSUE TITLE
Better error when function is undefined instaead of `Error: Incorrect type of the validation function.`

### ISSUE THREAD

**Issue by AleksandarDev** [Enterwell/react-form-validation#38]
- v1.11.1

```js
import { isNotEmptyString } from '@enterwell/react-form-validation';
```

![image](https://user-images.githubusercontent.com/643171/160499488-c03aa3b3-8ce4-49ab-a5a5-6c08547914dd.png)

### Suggestion

Maybe offer better message when validation function is undefined

**Comment by radovix** [comment]
So the bug is nonprecise error message or?

**Comment by AleksandarDev** [comment]
@radovix yep yep. When I created it I though it was same error fixed in 1.11.10 but then realized I had a typo. It would be nice there's different error for undefined validation function. 🙂
